hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/AbstractLedgerEvmWorldUpdater.java

```diff
@@ -16,23 +16,55 @@
 package com.hedera.node.app.service.evm.store.contracts;
 
 import com.hedera.node.app.service.evm.accounts.AccountAccessor;
+import com.hedera.node.app.service.evm.store.models.UpdatedHederaEvmAccount;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 import org.hyperledger.besu.datatypes.Address;
 import org.hyperledger.besu.datatypes.Wei;
 import org.hyperledger.besu.evm.account.Account;
 import org.hyperledger.besu.evm.account.EvmAccount;
 import org.hyperledger.besu.evm.worldstate.WorldUpdater;
+import org.hyperledger.besu.evm.worldstate.WorldView;
+import org.hyperledger.besu.evm.worldstate.WrappedEvmAccount;
 
-public abstract class AbstractLedgerEvmWorldUpdater implements WorldUpdater {
+public abstract class AbstractLedgerEvmWorldUpdater<W extends WorldView, A extends Account>
+        implements WorldUpdater {
 
+    protected final W world;
     protected final AccountAccessor accountAccessor;
+    protected Map<Address, UpdatedHederaEvmAccount<A>> updatedAccounts = new HashMap<>();
```

Comment:
We can use generic type and make the `updatedAccounts` map common for `AbstractLedgerEvmWorldUpdater` and `AbstractLedgerWorldUpdater`

Reply:
TODO - to be resolved in a future PR